### ISSUE TITLE
Any thoughts about having distributed tracer such as Jaeger 

### ISSUE THREAD

**Issue by liangjun-jiang** [linkedin/datahub#1809]
Jaeger is a distributed tracer tool. https://www.jaegertracing.io/
What do you think to have this global tracing feature to `Monitor and troubleshoot transactions in complex distributed systems`?

I understand, most likely, we deploy Datahub into a production environment, we would need something like that. Any thoughts about having it for an open source project? 
Definitely it will increase another complexity in terms of setting up Jaeger server to collecting & presenting those spans (logs)

**Comment by mars-lan** [comment]
Nice suggestion. I'm not super familiar with this space but is there any sort of standard emerging from the myriad of distributed log collection frameworks? Does docker or k8s offer any built-in support?

**Comment by shakti-garg-saxo** [comment]
@mars-lan For distributed tracing, I have found OpenTracing as the de-facto standard being adopted for obvious reasons(than non-efficient zipkins and spring-cloud-sleuth). 
For its implementations, Jaegar and Datadog(commercial) are most stable and mature implementations I have found so far(https://opentracing.io/docs/overview/tracers/)

Regarding with java integration, this is the library I have used before. (https://github.com/opentracing-contrib/java-kafka-client)

Apologies for late reply. Thanks.
